### DISCUSSION TITLE
(non-issue) Shiny Integration with iOS - DidFinishLaunching AppDelegate never called.

### DISCUSSION THREAD

**Discussion by csharpcoder198** [shinyorg/shiny#1013] [Ideas]
During this Sprint I had the Task of integrating Shiny into our existing App. A standard Xamarin App that has the standard 3 projects.

- Head Project (Done with Shiny 2.5.1 inclusion)
- Android Project (Done with Shiny 2.5.1 inclusion)
- iOS Project (Does not Launch - DidFinishLaunching() is never reached.)

I have read the Docs and studied in particular the GeoFencing Sample. I got the Head Project working against the Android Project just not the iOS Project. 

The iOS issue is the application steps out of the ios (Main.cs) and is never heard from again. No exception and never reaches the AppDelegate.DidFinishLaunching()

The master branch (pre shiny) to which this PR corresponds to does work in iOS and the AppDelegate.DidFinishLaunching is called.

I have a pull request between a master branch that does launch the iOS and Android project and a branch that I hope to integrate with Shiny 2.5.1 Integrated. Branch is named GeoFenceIntegration

[](https://github.com/csharpcoder198/GA360-APP/pull/1)

[Pull request is here!](https://github.com/csharpcoder198/GA360-APP/pull/1)

[Files Tab may be helpful (24 changes)](https://github.com/csharpcoder198/GA360-APP/pull/1/files#diff-59522cb4fc789bac69b0cc15669b31ed3c748069ec0892b9d3c8f8ee7ec7454b)

Any ideas?

**Comment by aritchie** [comment]
Please note that well I appreciate the sponsorship, you still need to go through the proper issue channels and provide SMALL reproducible samples.  The current geofencing sample is running fine.  Try turning off any linking and make sure all of the necessary services are being registered with Shiny.  There will be a crash log if something is missing.  If you can provide a small reproducible sample, I'm happy to take a look.

Also note: Mark Wardell (https://github.com/BicycleMark) was banned from here for being a rude.  I see the commits on this other project under that name.  Be warned that I don't care about a sponsorship.  Be polite, be concise, be to the point, or you'll be removed again Mark.

**Reply by csharpcoder198** [reply]
I am trying here. Thanks. Politeness is important. See what I can do.

**Reply by aritchie** [reply]
Your issue has nothing to do with shiny.  https://github.com/csharpcoder198/GA360-APP/blob/db1d66b9a82b23f027acc3a8748507c59e7be78f/GA360.iOS/Main.cs#L14

You have static main within static main.
